style.css

```diff
@@ -1,24 +1,24 @@
-html {
-  font-size: 16px;
-}
+@media (min-width: 375px) {
+  html {
+    font-size: 16px;
+  }
 
-a {
-  text-decoration: none;
-}
```

Comment:
- [ ] Dear @samuelkinyanjui, While you did a great job making improvements to your page there are still some details to be considered. And to start with, You have not used the right front from the template which is `Poppins` for your texts. 
- [ ] Besides that the sizing is not the same as the original design. Your heading text is a bit too much and your paragraph texts are too small. Please be reminded you can find all the font families and font sizes from the [Figma template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?type=design&node-id=39-122&t=mcwlqJqPiWDZETtj-0)
- [Optional] Though it has not been mentioned earlier, I see your background image is not taking the whole width of the page as depicted in the original design.  
- [ ] Lastly, As I have mentioned in my previous comments, Please take a look at the sizing of your social media icons. These are minor changes, but we need to see an almost identical design in our final product.  

### Your design 
![image](https://github.com/samuelkinyanjui/Portfolio-mobile-first/assets/69077061/cafa5da2-1dc0-42d3-8b79-03ab21692552)
### original design 
![image](https://github.com/samuelkinyanjui/Portfolio-mobile-first/assets/69077061/fc894701-980e-45d7-9d25-d38a65872290)

Reply:
Hi Can i call you right now?on slack

Reply:
front for the template which is Poppins is failing linter test and it is being flagged

Reply:
Sorry for the late replay @samuelkinyanjui. I notice you got it approved. Congrats!!